kwargs.jl

```diff
@@ -5,14 +5,23 @@ import Pkg
 include(joinpath(@__DIR__, "autodetect-dependabot.jl"))
 
 function kwargs(; coverage::Bool,
-                  force_latest_compatible_version::Union{Bool, Symbol})
+                  force_latest_compatible_version::Union{Bool, Symbol},
+                  julia_args::Union{Cmd, AbstractVector{<:AbstractString}}=``)
```

Comment:
I guess `Cmd` isn't used in practice as `kwargs` is only called from the driver script?

```suggestion
                  julia_args::AbstractVector{<:AbstractString}}=String[])
```

Reply:
Seems so! Sorry for the delay

Reply:
All good!  I must admit, I forgot about this completely and can't even remember which package I wanted it for :-)